Assets/Scripts/Throw/NextColor.cs

```diff
@@ -63,4 +72,20 @@ private ColorSO GetRandomColorSO()
     {
         return _gameManager.GetRandomColorSO();
     }
+
+    private ColorSO ThreeColors()
+    {
+   
+        if( _gameManager._ColorChangeRand==true)
```

Comment:
Can simplify by removing `==true`, since the `if` already check if the expression is true.